.projenrc.js

```diff
@@ -112,7 +112,16 @@ const project = new CustomTypescriptProject({
      */
     // for serializer testing
     "uuid",
+    "@swc/cli",
+    "@swc/core",
+    "@swc/register",
+    "@swc/jest",
+    "swc-closure@file:../swc-closure",
```

Comment:
TODO: consume the WASM from NPM

@thantos we need to decide on what to call the package, not sure how i feel about swc-closure

Reply:
`swc-functionless`? isn't it bound to functionless AST right now?

Reply:
Functionless ast is general though?

Reply:
@functionless/swc is probably fine?

Reply:
Went with @functionless/ast-reflection